css/style.css

```diff
@@ -33,10 +39,16 @@ body {
 
 main {
   margin-top: 70px;
+  background-image: url("../img/icons/circle-2-header.svg");
+  background-repeat: no-repeat;
+  background-position-x: 550px;
+  background-position-y: 185px;
+  background-size: 220px;
 }
 
 .jumbotron,
-.links {
+.links,
+.card-section {
```

Comment:
- [ ] Please use the right margins specified in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27).
- [ ] Please use the right fonts that are used in the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27).

Reply:
Effected changes here too